### ISSUE TITLE
Keyboard navigation does not work when direction is set to "right to left" 

### ISSUE THREAD

**Issue by brennanho** [tajo/react-range#158]
Looks like keyboard navigation works fine for "Left to right" but when you change direction to "right to left", the keyboard events do not reverse. For example, pressing the left arrow key on "right to left" should go right but instead it goes left. I'm guessing we can just reverse keyboard keys when the direction changes to "right to left"?

**Comment by tajo** [comment]
repro https://stackblitz.com/edit/react-range-edit?file=src%2FApp.tsx?
